<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcFunDownloader</class>
 <widget class="QMainWindow" name="AcFunDownloader">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AcFunDownloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>260</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg/ac.png);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#EE4000;
border-radius:5px;</string>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mini_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#6DDF6D;
border-radius:5px;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="net_act_layout" stretch="8,1,1">
       <item>
        <widget class="QLineEdit" name="input_box">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#00466B;
    border-radius:5px;
    background: transparent;</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>输入ac号或视频链接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:none;
    border:1px solid #C0DCF2;
    color:#00466B;
    padding:5px;
    min-height:20px;
    border-radius:5px;
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
    background: transparent;</string>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:none;
    border:1px solid #C0DCF2;
    color:#00466B;
    padding:5px;
    min-height:20px;
    border-radius:5px;
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
    background: transparent;</string>
         </property>
         <property name="text">
          <string>下载</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,3,3">
       <item>
        <widget class="QPushButton" name="fold_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:none;
    border:1px solid #C0DCF2;
    color:#00466B;
    padding:5px;
    min-height:20px;
    border-radius:5px;
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
    background: transparent;</string>
         </property>
         <property name="text">
          <string>文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:none;
    border:1px solid #C0DCF2;
    color:#00466B;
    padding:5px;
    min-height:20px;
    border-radius:5px;
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
    background: transparent;</string>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:none;
    border:1px solid #C0DCF2;
    color:#00466B;
    padding:5px;
    min-height:20px;
    border-radius:5px;
    background: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
    background: transparent;</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table_widget">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid #FFFFFF;
    color: black;
    border-radius:5px;
    background: transparent;</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <column>
        <property name="text">
         <string>新建列</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>新建列</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_bar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid #FFFFFF;
    height:20;
    text-align: center;
    color:black;
    border-radius:5px;
    background: transparent;</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AcFun - 认真你就输啦 (・ω・)ノ- ( ゜- ゜)つロ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
